packages/cfpb-layout/src/organisms/text-introduction.less

```diff
@@ -6,5 +6,6 @@
   & > * {
     margin-top: 0;
     margin-bottom: 15px;
+    max-width: 670px;
```

Comment:
This should be `max-width: 41.875rem`, which is how we set it in several other places. Aside—we should consolidate these max-width settings some time…

Reply:
Updated, thanks for catching that.